index.html

```diff
@@ -4554,10 +4554,9 @@ <h5>Link text</h5>
  the following steps need to be completed:
 
 <ol>
- <li><p>Let <var>elements</var> be the result of
-  calling <a>querySelectorAll</a>, with argument <a><code>a</code>
-  elements</a>, with the <a>context object</a> equal to the
-  <var>start node</var>. If this throws an exception,
+ <li><p>Let <var>elements</var> be the result of calling
+  <a>querySelectorAll</a> with <var>start node</var> as <a>this</a>
+  and "<code>a</code>" as the argument. If this throws an exception,
```

Comment:
This and the following similar change could be breaking what is intended by the spec. Unfortunately I can't find any tests in WPT involving SVG, so it would be some work to test if current WebDriver implementations would filter out non-HTMLAnchorElement values as the original text might suggest (but not correctly define).

Leaving this comment to see if others have thoughts before I dig more.

Reply:
I would expect it would only consider elements that represent links; getting some `foo:a` elements in a made up namespace. However `svg:a` is also a link, so that seems like it perhaps ought to be considered? idk what existing implementations do here.

Reply:
I tried to find out, and it seems like ChromeDriver implements this using a large snippet of JS from Selenium. I think this is what ends up being used:
https://github.com/SeleniumHQ/selenium/blob/64447d4b03f6986337d1ca8d8b6476653570bcc1/javascript/atoms/locators/link_text.js#L76

So that boils down to `root.querySelectorAll("a")` and I can't see anything that tries to filter to `HTMLAnchorElement` and `SVGAElement`. So I guess what I wrote here ends up matching at least chromedriver better than the original text.

I guess this should be tested though...

Reply:
https://searchfox.org/mozilla-central/source/testing/marionette/element.js#452 looks like it boils down to the same thing. I'm prepared to say this is stilly an unintentional, but probably not worth changing at this point.